content/posts/irods-development-update-november-2022.md

```diff
@@ -9,16 +9,20 @@ We went to Supercomputing 2022!
 
 We got a chance to talk with Consortium members in person and there was a lot of booth traffic. Overall, it was a success!
 
-Okay, enough about Supercomputing. Let's get into the update.
+Also, TRiRODS will be happening on Wednesday, December 7 at 2pm ET. Martin Schobben will be talking about the new **rirods** projects he's been working. You can read more about that at [https://irods.org/trirods/](https://irods.org/trirods/).
```

Comment:
"been working" -> "been working on"

"read more about that at" -> "read more at"

Reply:
Done